### DISCUSSION TITLE
Práctica módulo CSS [Casi terminada]

### DISCUSSION THREAD

**Discussion by JoseEduardoVelazquezCN** [Launch-X-Latam/MisionFrontEnd#323] [Show and tell]
Hola buenas noches a todos está es mi repositorio con la práctica del módulo de css 

Espero sus críticas constructivas y comentarios 🐧 

Link a mi repositorio: https://github.com/JoseEduardoVelazquezCN/CSS-Practicas-Laucnch-X-Mision-FrontEnd-

Link del despliegue: https://vacinnation-jevcn.netlify.app/

Hay un componente que me está desbordando la página en el diseño responsive  (algún día lo encontraré)

**Comment by JoniP-IV** [comment]
No sé a que te referís con "desbordando" pero ahí estuve inspeccionando y lo que veo es que hay un problema con lo que llamas "modal", creo que está tomando todo el ancho y alto de la sección, cuando en realidad solo debería ocupar su espacio verdad?

**Reply by JoseEduardoVelazquezCN** [reply]
Con desbordado me refiero a que si te fijas se puede desplazar horizontalmente y todos los componentes llegan a un tope pero hay un desplazamiento horizontal que no debería estar 
Pero si creo que hay un problema con el elemento con nombre modal, colocaré borde a todo para ver cuál es el que ocupa más espacio del que debería

**Reply by JoniP-IV** [reply]
Siii entonces estamos en lo correcto, hay un problema con ese elemento, creo que lo que está dificultando el posicionamiento es la propiedad "position: absolute"... esas propiedades te cambian todo el layout generalmente..

**Reply by OscarJG20** [reply]
Sino me equivoco, en realidad el modal está bien, lo que mencionan sobre ese contenedor y que ocupa demasiado es porque le puso un margin-top de 760px, me imagino que para mantener el estilo, ahí no veo un desbordamiento.

**Comment by OscarJG20** [comment]
Hola Jose!
No sé si ya encontraste la solución, pero he encontrado tu desbordamiento. Yo creí que era por los modals o algo más, realmente el error solo está en el título de "Vaccination",
![image](https://user-images.githubusercontent.com/64516675/200030139-cd4df46c-ef93-462e-b6d2-06c047f33ed9.png)

Puedes quitar tu margin-left y poner un padding-left, te explico el error, el contenedor ".titulo" le das un 100%, eso está bien, pero a los elementos hijos (h1 donde tienes el problema) también les diste el 100% en width (no está mal), pero además le diste un margin-left, por eso se causa el desbordamiento, porque tanto el padre (".titulo") como el hijo ("h1") ya ocupan el 100% del width, poner un margin-left crea un espacio vacío entre ellos (desbordamiento), para obtener el mismo resultado y sin desbordamiento, solo cambia ese margin por un padding y listo!

Espero que te ayude, por cierto, tu web se ve bastante bien. :)

**Reply by JoseEduardoVelazquezCN** [reply]
acabo de revisar esto y si era eso, muchas gracias por el apoyo, ahoa si solo falta actualizarlo en el despliegue y listo..... Muchas gracias Oscar !!!!!
